### PR TITLE
add support for moog style filter

### PR DESCRIPTION
**Pull request by friedolino78**
GUI Part of 
https://github.com/zynaddsubfx/zynaddsubfx/pull/85

### CODE REVIEW

File: src/mruby-zest/qml/NumEntry.qml [hunk 1]
```diff
@@ -24,6 +24,26 @@ Widget {
 
     }
 
+    function changeMin(wantedMin)
+    {
+        if (self.value > wantedMin)
```
Comment (JohannesLorenz) [inline src/mruby-zest/qml/NumEntry.qml:29]: Copy-Paste-Error?

```suggestion
        if (self.value < wantedMin)
```

**Comment by JohannesLorenz** [comment]
I'm about to review the UI now.

**Comment by fundamental** [comment]
For the record I recall looking at the old PR and it looking fine, though I didn't execute the code. (a step which on the Comb Filter PR caused a few issues to appear).

**Review by JohannesLorenz — CHANGES_REQUESTED** [review]
Only one comment and one issue from testing:

Select a wave with rich harmonics (e.g. power wave), leave the cutoff at maximum, change analog filter type to HP1. Then, change the filter to moog. The moog type will be displayed as "LP", but you don't hear anything. That must be wrong, because a LP at maximum frequency should let the wave pass nearly unchanged. As soon as you click the "LP", it says it's actually "HP".

**Comment by JohannesLorenz** [comment]
Also, no really critical, but the spectrum diagram shows a sine wave if I change it from analog to moog.

**Comment by friedolino78** [comment]
> Also, not really critical, but the spectrum diagram shows a sine wave if I change it from analog to moog.

yeah, maybe for the first step I should better simply copy the analog plot.

**Pull request merged**